crates/stackable-telemetry/src/instrumentation/axum/mod.rs

```diff
@@ -363,11 +362,13 @@ pub trait SpanExt {
 
 impl SpanExt for Span {
     fn from_request(req: &Request, opt_in: bool) -> Self {
+        let _otel_context_guard = Context::current();
```

Comment:
question: I'm curious why this is needed. Shouldn't the following spans / events use the current context by default?

Reply:
Yes, but as you and I both saw, the context wasn't being carried across and the Axum route spans then had no parent and generated new trace IDs, thereby splitting one trace into two separate traces.

This was based on how it worked before #806 was merged, so I haven't gone back to see if it might not be needed now.

What do you suggest here?

Reply:
> What do you suggest here?

If you really want, you can test it again. In case we still need it, we should add a comment stating why we need it - so that we don't stumble over this in the future.

Reply:
It seems not needed.

Removed in d5d90bd0081d711002c87c15bb9d01ac1eb173fe